### PR TITLE
Cash flow statement does not work if there are no profit & loss expenses

### LINKED ISSUE #9098: Cash flow statement does not work if there are no profit & loss expenses
when selling items with no stock, cashflow statement does not show net profit & loss line under operating cashflow.

This is due to net_profit_loss = get_net_profit_loss(income, expense, period_list, filters.company) breaking when there are no expenses.

minor bug

### PR DESCRIPTION
**Pull request by tundebabzy**
Fixes #9098 

### Before
screenshot from 2017-06-13 23-34-53
screenshot from 2017-06-13 23-35-24

### After
screenshot from 2017-06-13 23-36-30
screenshot from 2017-06-13 23-36-45

### CODE REVIEW

**Comment by tundebabzy** [comment]
Wrong branch.
